pbj-core/pbj-runtime/src/test/java/com/hedera/pbj/runtime/io/DataTest.java

```diff
@@ -82,6 +82,10 @@ static Stream<Long> unsignedIntsTestCases() {
         return Stream.of(0,1,9,51,127,Integer.MAX_VALUE*2L).map(Number::longValue);
     }
 
+    static Stream<Integer> unsignedIntsTestCasesInts() {
+        return Stream.of(0,1,9,51,127,(int)Integer.MAX_VALUE*2L).map(Number::intValue);
+    }
```

Comment:
Is this related to the fix?

Reply:
This was just a test that was missing - there were for these, but I'll take it out. Thabnks! Good catch!